README.md

```diff
@@ -15,75 +15,72 @@
+You can find a complete walk through in our [Getting Started](./docs/getting-started.md).
 
-### Project Templates
+### FAQ
 
-Hops comes with a couple of different project templates you can use with `hops init --template`. If you do not explicitly select one, `hops init` defaults to `hops-template-react`.
+#### When is Hops the framework for you?
 
-| Package                                                                                     | NPM                                                                                                                   |
-| ------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
-| [Hops Template Minimal](https://github.com/xing/hops/tree/master/packages/template-minimal) | [![npm](https://img.shields.io/npm/v/hops-template-minimal.svg)](https://www.npmjs.com/package/hops-template-minimal) |
-| [Hops Template React](https://github.com/xing/hops/tree/master/packages/template-react)     | [![npm](https://img.shields.io/npm/v/hops-template-react.svg)](https://www.npmjs.com/package/hops-template-react)     |
-| [Hops Template GraphQL](https://github.com/xing/hops/tree/master/packages/template-graphql) | [![npm](https://img.shields.io/npm/v/hops-template-graphql.svg)](https://www.npmjs.com/package/hops-template-graphql) |
+When you want to share a consistent setting over more than one project. Either now or in the future. This applies to companies with more than one development team or when you constantly set up projects that should be consistent, e.g. training projects, tutorials, consultant work, etc.
 
-If, for example, you want to bootstrap your project to use both React and GraphQL, you will have to call `hops init` like this:
+#### When is Hops **not** the framework for you?
 
-```shell
-hops init my-graphql-project --template hops-template-graphql
-```
+If you are new to React and just want a small sample app or mainly want to learn, we recommend to get started with [Create React App](https://github.com/facebook/create-react-app). Once you are getting more serious about your application you can still switch to Hops.
 
-### General Usage
+#### Why not just use Create React App?
 
-Hops is quite modular, its components being independently published on npm. Its main building blocks, however, rely upon each other:
+[Create React App](https://github.com/facebook/create-react-app) is great when you want to experiment with or learn React. For simplicity it is neither configurable nor extensible and does not evolve with you when you are ready to create a production ready app. Hops on the other hand might be a bit harder to get started, but has the complete potential for even large scale applications.
```

Comment:
Should server-side rendering be mentioned here?

Reply:
Yes, I guess so. Will do.